lib/routes/create.js

```diff
@@ -34,7 +34,7 @@ createRoute.register = () => {
         const theirs = request.params.data
         theirResource = _.assign(
           {
-            id: uuid.v4(),
+            id: 0,
```

Comment:
This is 0 here as a "DEFAULT" value.
Putting the onus on the store handler (relattionaldb) to treat 0 = DEFAULT and if the value is 0, then generate new id, if not 0, then use it like a client provided id. 

see https://github.com/championswimmer/jsonapi-store-sequelize/commit/ee0b9f0303f9101577807e0dd8b720b8f67ab9d7#diff-8004fa36f21fa5eeb3c49b89106f6942R9 for example implementation